lib/ui/window.dart

```diff
@@ -286,6 +326,29 @@ class Locale {
     }
   }
 
+  /// The variants subtags for the locale.
+  ///
+  /// Each subtag is expected to be a valid Unicode Language Identifier variant
+  /// subtag that is listed in Unicode CLDR supplemental data:
+  /// http://unicode.org/cldr/latest/common/validity/variants.xml. Please see
+  /// constructor documentation.
+  Iterable<String> get variants {
+    if (_variants == null)
+      return const <String>[];
+    return _variants.split('-');
```

Comment:
this will return a new Iterable each time, which means `locale.variants == locale.variants` will return false. If we want to store the variants as a delimited string (which I guess is what both Android and iOS give us? o_O) then we should make this a method.

However, it seems weird to store the data in a half-encoded form. Should we not require people to provide us the variants pre-split?

Reply:
Done. Documented the fact that the calling code should not modify the List<String> it passes, as we then use this list object directly, wishing it was immutable.